tests/runner/test_runner.py

```diff
@@ -768,6 +769,13 @@ def test_from_flow_accepts_rrule(self):
 
         assert deployment.schedule.rrule == "FREQ=MINUTELY"
 
+    def test_from_flow_accepts_is_schedule_active(self):
```

Comment:
nit: It looks like there's an opportunity here to parameterize over `is_schedule_active` to test out the cases where it's explicitly set to `True`/`False`/`None` and the expected setting after doing `from_flow`/`from_entrypoint`. I'll defer to you whether you think those are code paths worth testing.